docs/en/index.rst

```diff
@@ -193,6 +193,7 @@ Migration names can follow any of the following patterns:
   specified table
 * (``/^(Alter)(.*)/``) Alters the specified table. An alias
   for CreateTable and AddField.
+* (``/^(Alter).*(?:From)(.*)/``) Alters fields from the specified table.
```

Comment:
Instead of `From` should this be `On`?

Reply:
Thank you very much. I will update and commit shortly.

Reply:
Mr, @markstory , I have updated and pushed new updates already. 

Hope to see you at CakeFest in Tokyo. :) I will attend in Conference.